findme-users/handler.py

```diff
@@ -75,7 +79,7 @@ def update_user():
 @app.get("/users/<user_id>")
 @tracer.capture_method
 @authorizer.requires_auth(app=app)
-def get_user(user_id: Annotated[int, Path(lt=999)]):
+def get_user(user_id: Annotated[str, Path(lt=999)]) -> UserDTO:
```

Comment:
here I think lt=999 should be removed
```suggestion
def get_user(user_id: Annotated[str, Path()]) -> UserDTO:
```